mne/io/eeglab/eeglab.py

```diff
@@ -734,7 +738,31 @@ def _bunchify(items):
     return items
 
 
-def _read_annotations_eeglab(eeg):
+def _read_annotations_eeglab(eeg, output_type=None):
```

Comment:
you should not use an Annotations to store onset in samples. it's hacking the semantics of Annotations object.

I know it's my mistake in the first place :)

Reply:
It's to be backward compatible, and change read_events afterwards in a diferent PR.